composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -35,9 +39,35 @@ fun main(rawArgs: Array<String>) {
         log("$args")
     }
 
+    val testPackage = getTestPackageFromTestApk(args.testApkPath)
```

Comment:
rename to `parseTestPackage(testApkFile = File(args.testApkPath))`?

Reply:
OK